ceph.spec.in

```diff
@@ -153,6 +156,15 @@ BuildRequires:  openldap-devel
 BuildRequires:  openssl-devel
 BuildRequires:  redhat-lsb-core
 BuildRequires:	Cython
```

Comment:
OpenSUSE has *python-Cython*, Fedora/RH/CentOS have *Cython*. I think all is correct here.

Yes, it is still needed to build Python 2 extension libraries.